src/components/calendar/calendar.test.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import Calendar from './calendar';
+import { CalendarIcon, ExpandIcon } from '../icons';
+
+describe('<Calendar />', () => {
```

Comment:
I feel this tests are not comprehensive enough. How do you test that this component has multiple ui classes
how do you test that it contains span, how do you test how many divs it has? how do you test for the text it contains?